src/Command/Environment/EnvironmentDrushCommand.php

```diff
@@ -127,12 +127,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $command = 'ssh' . $sshOptions . ' ' . escapeshellarg($sshUrl)
             . ' ' . escapeshellarg($sshDrushCommand);
 
-        if ($output->getVerbosity() >= OutputInterface::VERBOSITY_VERBOSE) {
-            $this->stdErr->writeln("Running command: <info>$command</info>");
-        }
+        $this->stdErr->writeln("Running command: <info>$command</info>", OutputInterface::VERBOSITY_VERBOSE);
 
-        passthru($command, $return_var);
+        $process = proc_open($command, [STDIN, STDOUT, STDERR], $pipes);
+        $status = proc_get_status($process);
+        proc_close($process);
```

Comment:
That looks wrong. `proc_get_status()` returns an array with a ton of stuff. What you want is just `$status = proc_close($process);`.